### PR TITLE
Add Swashbuckle, XML comments

### PR DESCRIPTION
**Pull request by sober607**
Added Swashbuckle, XML comments and sample request and responses to Secretaries controller #109

Sample:
[Img2](https://ibb.co/4NMscjL)
[Img1](https://ibb.co/P4TPZn8)

### CODE REVIEW

**Review by SnipeFrost — APPROVED** [review]

**Review by BohdanBBG — APPROVED** [review]

**Review by SerjKocherga — APPROVED** [review]

File: CharlieBackend.Api/Controllers/SecretariesController.cs [hunk 1]
```diff
@@ -24,6 +24,15 @@ public SecretariesController(ISecretaryService secretaryService, IAccountService
...
         }
 
+        /// <summary>
+        /// Gives secretary role to account and creates secretary entity
+        /// </summary>
+        /// <param name="accountId">Account id to approve</param>
+        /// <response code="200">Successful secretary entity create</response>
+        /// <response code="400">Account already assigned to secretary</response>
+        /// <response code="404">Acccount not found</response>
+        /// <response code="500">Can not assign account to secretary</response>
```
Comment (whereismyroot) [inline CharlieBackend.Api/Controllers/SecretariesController.cs:34]: 500 error happens when we didn't handle some case and something went wrong on the server. We do not need to declare it in our contract.

File: CharlieBackend.Api/CharlieBackend.Api.csproj [hunk 2]
```diff
@@ -23,15 +28,18 @@
...
     <PackageReference Include="DotNetEnv" Version="1.4.0" />
     
+    <PackageReference Include="Swashbuckle.AspNetCore.Filters" Version="6.0.1" />
+    
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\CharlieBackend.Business\CharlieBackend.Business.csproj" />
     <ProjectReference Include="..\CharlieBackend.Core\CharlieBackend.Core.csproj" />
+    <ProjectReference Include="..\CharlieBackend.Library\CharlieBackend.Library.csproj" />
```
Comment (whereismyroot) [inline CharlieBackend.Api/CharlieBackend.Api.csproj:42]: Why do we need separate project for examples? Also it is pretty generic name for swagger-specific classes. Please remove this project and place examples folder in CharlieBackend.Api.

File: CharlieBackend.Api/CharlieBackend.Api.csproj [hunk 3]
```diff
@@ -5,6 +5,11 @@
     <UserSecretsId>77c89e00-81d7-4d64-9c07-8a105a6066bb</UserSecretsId>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>$(NoWarn);1591</NoWarn>
```
Comment (whereismyroot) [inline CharlieBackend.Api/CharlieBackend.Api.csproj:10]: do not ignore this warning. It will help to document all public facing properties and API's

**Review by whereismyroot — CHANGES_REQUESTED** [review]
nice work, but need some changes

File: CharlieBackend.Api/Controllers/SecretariesController.cs (hunk 1 above)
Comment (sober607) [reply CharlieBackend.Api/Controllers/SecretariesController.cs:34]: Ok

File: CharlieBackend.Api/CharlieBackend.Api.csproj (hunk 2 above)
Comment (sober607) [reply CharlieBackend.Api/CharlieBackend.Api.csproj:42]: This was recommendation of Mentor on 20.11 meeting.

Ok.

File: CharlieBackend.Api/CharlieBackend.Api.csproj (hunk 3 above)
Comment (sober607) [reply CharlieBackend.Api/CharlieBackend.Api.csproj:10]: Ok.

**Review by whereismyroot — APPROVED** [review]

**Pull request merged**